src/smolagents/utils.py

```diff
@@ -187,6 +187,7 @@ def parse_code_blobs(text: str) -> str:
         ValueError: If no valid code block is found in the text.
     """
     pattern = r"```(?:py|python)?\s*\n(.*?)\n```"
+    text = text.split("Code:")[-1]
```

Comment:
This change assumes that only a "Code:" marker appears in the model output.

@aymeric-roucher do you think this is a sensible assumption? Alternative assumptions?